x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.ts

```diff
@@ -157,14 +175,20 @@ export function useBulkActions({
   useBulkActionsConfig = () => [],
 }: BulkActionsProps): UseBulkActions {
   const [bulkActionsState, updateBulkActionsState] = useContext(BulkActionsContext);
-  const configBulkActions = useBulkActionsConfig(query);
+  const configBulkActionPanels = useBulkActionsConfig(query);
 
   const clearSelection = () => {
     updateBulkActionsState({ action: BulkActionsVerbs.clear });
   };
   const caseBulkActions = useBulkAddToCaseActions({ casesConfig, refresh, clearSelection });
 
-  const bulkActions = [...configBulkActions, ...caseBulkActions];
+  const bulkActions =
+    caseBulkActions.length !== 0
```

Comment:
another condition we should test

Reply:
yes this hook also probably runs too often as well, both the old way and new way.